packages/api/src/error.ts

```diff
@@ -110,6 +111,6 @@ export class DuplicatePageSlugError extends ApolloError {
 
 export class CommentLengthError extends ApolloError {
   constructor() {
-    super(`Comment length should not exceed 1000 characters.`, ErrorCode.CommentLengthError)
+    super(`Comment length should not exceed ` + MAX_COMMENT_LENGTH + ` characters.`, ErrorCode.CommentLengthError)
```

Comment:
With [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) you can write the line like this `.... should not exceed ${MAX_COMMENT_LENGTH} characters....`